osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -165,13 +165,6 @@ public int RulesetID
             }
         }
 
-        [Ignored]
-        public BeatmapDifficulty BaseDifficulty
-        {
-            get => Difficulty;
-            set => Difficulty = value;
-        }
```

Comment:
Do we care about external rulesets potentially still having references to this? As in would we want this to be around for a bit more, but just marked as obsolete, or is this a hard blocker for further work?

Reply:
do you have an example of this being used by a rule set?

Reply:
One example is similar to the usage which was changed here: https://github.com/taulazer/tau/blob/master/osu.Game.Rulesets.Tau/Beatmaps/TauBeatmapConverter.cs

Reply:
Fair. Will add back as `Obsolete` for this case.